### PR TITLE
Add ability to fail report if no data (#8)

### PR DESCRIPTION
**Pull request by baev**

### CODE REVIEW

File: src/main/java/ru/yandex/qatools/allure/report/AllureReportMojo.java [hunk 1]
```diff
@@ -98,11 +111,15 @@ protected void executeReport(Locale locale) throws MavenReportException {
         try {
             Aether aether = Aether.aether(repoSystem, repoSession, projectRepos);
             AllureReportBuilder builder = new AllureReportBuilder(reportVersion, outputDirectory, aether);
-            builder.setClassLoader(Thread.currentThread().getContextClassLoader());
 
             getLog().info("Generate report to " + outputDirectory);
             builder.processResults(reportDirectories);
 
+            if (failReportIfEmpty && isReportEmpty()) {
+                throw new MavenReportException("There are no test cases info in given result" +
```
Comment (lanwen) [inline src/main/java/ru/yandex/qatools/allure/report/AllureReportMojo.java]: which directory is given?

Comment (baev) [inline src/main/java/ru/yandex/qatools/allure/report/AllureReportMojo.java]: `#getReportDirectories`

Comment (lanwen) [inline src/main/java/ru/yandex/qatools/allure/report/AllureReportMojo.java]: think it should be in exception

Comment (baev) [inline src/main/java/ru/yandex/qatools/allure/report/AllureReportMojo.java]: @lanwen now all found directories will be logged.

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Pull request merged**